sympy/core/numbers.py

```diff
@@ -4276,41 +4278,113 @@ def all_close(expr1, expr2, rtol=1e-5, atol=1e-8):
     False
     >>> all_close(expr1, expr2)
     True
+
+    Identities are automatically supplied:
+
+    >>> all_close(x, x + 1e-10)
+    True
+    >>> all_close(x, 1.0*x)
+    True
+    >>> all_close(x, 1.0*x + 1e-10)
+    True
```

Comment:
This is definitely not what I wanted when creating this function or whenever I have used it. Its purpose was to be used in tests where these sorts of differences should generally show up as test failures.

That now means that I want a different function for use in tests because this one does not do what is actually needed there any more.